oracle-plugin/src/e2e-test/java/io.cdap.plugin/common.stepsdesign/TestSetupHooks.java

```diff
@@ -45,6 +45,74 @@ public static void createTables() throws SQLException, ClassNotFoundException {
                                    PluginPropertyUtils.pluginProp("schema"));
   }
 
+  @Before(order = 1, value = "@ORACLE_SOURCE_DATATYPES_TEST")
+  public static void setSelectQueryForDatatypes1() {
```

Comment:
Ideally the function names should not contain integers, do use a more descriptive names.

Reply:
Changed the names to be more descriptive.